### PR TITLE
[docs] Improve the TS code examples

### PR DESCRIPTION
**Pull request by jjleng**
### Description
Improve the code examples by using TS SDK. We don't want developers to copy and paste the code to submit txns with JSON payloads.

### Test Plan
Tested every code example manually

---
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/2365)

### CODE REVIEW

**Review by gregnazario — COMMENTED** [review]
There's a lot for me to go through here on the fixes on the TS examples, but it looks good!  Did you completely replace the one of SDKs with the TS SDK?

File: developer-docs-site/docs/tutorials/first-coin.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ For Typescript:
 - Download the [example project](https://github.com/aptos-labs/aptos-core/tree/main/developer-docs-site/static/examples/typescript).
 - Open your favorite terminal and navigate to where you downloaded the above example project
 - Install the required libraries: `yarn install`
-- Execute the example: `yarn first_coin MoonCoin.mv`
+- Execute the example: `yarn first_coin moon_coin.mv`
```
[the remark this answers is not in the document]
Comment (jjleng) [reply developer-docs-site/docs/tutorials/first-coin.md:89]: Screen Shot 2022-07-29 at 10 10 12 PM

[the remark this answers is not in the document]
Comment (jjleng) [reply developer-docs-site/docs/tutorials/first-coin.md:89]: It becomes moon_coin.mv after renaming the module name

**Comment by jjleng** [comment]
Completely updated TS examples to use the TS SDK.

File: developer-docs-site/docs/tutorials/first-coin.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (gregnazario) [reply developer-docs-site/docs/tutorials/first-coin.md:89]: Ah, yes it's the compiled module not the source

**Review by gregnazario — APPROVED** [review]

**Pull request merged**